packages/dds/tree/src/tableSchema.ts

```diff
@@ -1226,7 +1213,7 @@ export namespace TableSchema {
 
 		/**
 		 * Removes the specified column from the table.
-		 * @param column - The {@link IColumn | column} or {@link IColumn.id | column ID} to remove.
+		 * @param column - The {@link TableSchema.IColumn | column} or {@link TableSchema.IColumn.id | column ID} to remove.
```

Comment:
These links were broken. I'm honestly really surprised this got past CI - doing a local API-Extractor build failed on these.